test/BUILD.bazel

```diff
@@ -0,0 +1,44 @@
+load("@rules_cc//cc:defs.bzl", "cc_library", "cc_test")
+load("@bazel_skylib//lib:selects.bzl", "selects")
+
+# buildifier: keep sorted
+_TESTS = [
+    "container",
+    "core",
+    "entity",
+    "version",
+]
```

Comment:
I didn't add all the tests, because I didn't want it to be a burden to update this. Since EnTT is already well covered test-wise with CMake I didn't see it as necessary. Especially since the bazel BUILD files previously did not even have tests.

Reply:
Maybe meta rather than container, since it's probably the second most used submodule? Also signal looks like a good candidate.

Reply:
the meta tests are failing 😕 
https://github.com/skypjack/entt/actions/runs/6027899063/job/16354072478?pr=1057

I browsered through the cmake files to see if there's any special flags that I'm not passing to bazel, but I didn't see anything.

Is there something obvious I'm missing that you can think of?

Reply:
No idea actually. Are you compiling tests as shared library? In this case, the context could be different and the tests would fail.

Reply:
No I'm not, but that gives me some clues. I'll look into it soon.

Reply:
I finally got around to looking at this. I didn't understand that the meta tests needed to be compiled separately like the CMake config was doing. That fixed it!

Reply:
Oh, did you compile all tests in a single executable before? If this is the case, I strongly suggest to split them as I do in the cmake file then.
I want to _clean up_ things and make it possible to compile everything at once but I never found the time to work on this task yet. I'm sorry for the inconvenience.
Let me know if it's ok to merge or if we need to split it a little further. Thanks. 👍

Reply:
yes they're all separate now! Everything is good to go.